qa/os/src/test/java/org/elasticsearch/packaging/test/PackageTests.java

```diff
@@ -190,7 +190,7 @@ public void test50Remove() throws Exception {
             } else {
 
                 final Result versionResult = sh.run("systemctl --version");
-                final Matcher matcher = Pattern.compile("^systemd (\\d+)\n").matcher(versionResult.stdout);
+                final Matcher matcher = Pattern.compile("^systemd (\\d+)\\s").matcher(versionResult.stdout);
```

Comment:
Maybe just remove the newline? We just care about capturing the number in parens, and don't care about anything after that.